test/mocks/mockCourseAnalysis.js

```diff
@@ -9,6 +9,7 @@ const mockCourseAnalysis = (semester = '20172', roundIdList = '1') => ({
   pmFileName: 'pm-EI1220HT2017_1.pdf',
   analysisFileName: 'analysis-EI1220HT2017_1.pdf', // prev file from api
   roundIdList,
```

Comment:
`roundIdList` has to be removed, in order to be sure tests work

Reply:
roundIdList was removed from everywhere